crates/orchestrator/src/spawner.rs

```diff
@@ -116,6 +116,12 @@ where
         relay_data_path: &relay_data_path, // TODO: get from provider
         use_wrapper: false,                // TODO: get from provider
         bootnode_addr: ctx.bootnodes_addr.clone(),
+        // IFF the provider require an image (e.g k8s) we should use the default ports in the cmd.
+        use_default_ports_in_cmd: if ctx.ns.capabilities().requires_image {
+            true
+        } else {
+            false
+        },
```

Comment:
```requires_image``` is already a boolean:

```suggestion
        use_default_ports_in_cmd: ctx.ns.capabilities().requires_image,
```